lib/models/playing_cards.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+enum CardSuit {
+  spades,
+  hearts,
+  diamonds,
+  clubs,
+}
+
+enum CardType {
+  one,
+  two,
+  three,
+  four,
+  five,
+  six,
+  eight,
+  nine,
+  ten,
+  jack,
+  queen,
+  king
+}
+
+enum CardColor {
+  red,
+  black,
+}
+
+// Simple playing card model
+class PlayingCard {
+  CardSuit cardSuit;
+  CardType cardType;
+  bool opened;
+
```

Comment:
A card also needs to have a field -> "name" which shows the name of the player to which it belongs to (currently) and it is a required field.